worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitAdapter.java

```diff
@@ -304,6 +306,27 @@ public static GameMode adapt(org.bukkit.GameMode gameMode) {
         return GameModes.get(gameMode.name().toLowerCase());
     }
 
+    /**
+     * Create a WorldEdit BiomeType from a Bukkit one.
+     *
+     * @param biome Bukkit Biome
+     * @return WorldEdit BiomeType
+     */
+    public static BiomeType adapt(Biome biome) {
+        return BiomeTypes.get(biome.name().toLowerCase());
+    }
+
+    public static Biome adapt(BiomeType biomeType) {
+        if (!biomeType.getId().startsWith("minecraft:")) {
+            throw new IllegalArgumentException("Bukkit only supports vanilla biomes");
+        }
+        try {
+            return Biome.valueOf(biomeType.getId().substring(10).toUpperCase());
+        } catch (IllegalArgumentException e) {
+            return null;
```

Comment:
This is weird -- either we should return `null` in all error cases, or throw an error. You do the former here, and the latter above.

Reply:
This is the only one using the Enum#valueOf method, which throws an exception. I don't believe the others do?

Reply:
Oh, weird. Looks like it'd be consistent then.